Doc/whatsnew/3.8.rst

```diff
@@ -149,6 +149,22 @@ extensions compiled in release mode and for C extensions compiled with the
 stable ABI.
 (Contributed by Victor Stinner in :issue:`36722`.)
 
+To embed Python into an application, a new ``--embed`` option must be passed to
+``python3-config --libs --embed`` to get ``-lpython3.8`` (link the application
+to libpython). To support Python 3.7 and older, try ``python3-config --libs``
+(without ``--embed``) on failure.
+
+Add a pkg-config ``python-3.8-embed`` module to embed Python into an
+application: ``pkg-config python-3.8-embed --libs`` includes ``-lpython3.8``.
+To support Python 3.7 and older, try ``pkg-config python-X.Y --libs`` (without
+``-embed``) on failure (replace ``X.Y`` with the Python version).
```

Comment:
To support both 3.8 and older, try `pkg-config python-X.Y-embed --libs` first and fallback to `pkg-config python-X.Y --libs` (without `-embed`) if the previous command fails (replace `X.Y` with the Python version).

Reply:
I picked your sentence, thanks.